lockfile.json

```diff
@@ -362,6 +362,12 @@
   "nickel": {
     "revision": "b759233581fd8bae239e905c67a9ba453205da78"
   },
+  "nim": {
+    "revision": "388b53f808eb162963cfe99bd2d1ed83c8b5f8a1"
```

Comment:
If you don't mind, please bump this to `35c0245439947e66239c1d59ea28b3fb0bba3d4c`.

I just tagged 0.1.0 for this occasion :partying_face:

Reply:
This file is autonated, so the revision here is not important (as long as the queries match). The only important thing for CI at this point is that there _is_ an entry.